pkg/registry/apps/deployment/storage/storage.go

```diff
@@ -137,6 +138,20 @@ type RollbackREST struct {
 	store *genericregistry.Store
 }
 
+// ProducesMIMETypes returns a list of the MIME types the specified HTTP verb (GET, POST, DELETE,
+// PATCH) can respond with.
+func (r *RollbackREST) ProducesMIMETypes(verb string) []string {
+	return nil
```

Comment:
What does nil mean here? Hopefully "use the default"?

Reply:
Yes. The returned list gets appended to default media types, so nil means using the default
https://github.com/kubernetes/kubernetes/blob/972a74e23805d3d75bc68ddf5bf9c409e8f9b294/staging/src/k8s.io/apiserver/pkg/endpoints/installer.go#L943-L945